components/SiteHeader.tsx

```diff
@@ -1,21 +1,46 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Link from 'next/link'
+import classNames from 'classnames'
 import LogoIcon from '@/components/svg/LogoIcon'
 import ButtonLink from '@/components/ButtonLink'
+import HamburgerMenu from '@/components/HamburgerMenu'
 
 const AppHeader: React.FC = () => {
+  const [showMobileNav, setShowMobileNav] = useState(false)
+
+  function toggleMobileNav() {
+    setShowMobileNav(!showMobileNav)
+  }
+
   return (
     <div className='z-40 text-primary'>
       <header className='flex items-center justify-between p-2 mx-auto max-w-7xl sm:px-6 lg:px-8'>
-        <div className='flex-none mr-4'>
+        <div className='relative z-50 flex-none mr-4'>
           <Link href='/'>
             <a className='text-primary hover:text-gray-900 dark:hover:text-white' aria-label='Go to dashboard'>
               <LogoIcon scale={1.2} />
             </a>
           </Link>
         </div>
-        <nav className='grid items-center grid-flow-col gap-4'>
-          <div className='items-center hidden text-sm font-semibold grid-flow-col gap-4 md:grid'>
+        <div
+          className={classNames('transition duration-300 fixed z-10 inset-0 transition-opacity bg-blur', {
```

Comment:
Also super minor and not a real problem a customer would see but if you go from mobile to big screen with the nav menu open the mask sticks around

```suggestion
          className={classNames('md:hidden transition duration-300 fixed z-10 inset-0 transition-opacity bg-blur', {
```